java/ql/lib/change-notes/2024-06-13-kotlin-qltest-support.md

```diff
@@ -0,0 +1,4 @@
+---
+category: fix
+---
+* Support of `codeql test run` for Kotlin sources has been fixed.
```

Comment:
@igfoo does this sound right? Or would you prefer another wording?

Reply:
I'd probably have written "Support for".